docs/aws_freertos_port.md

```diff
@@ -0,0 +1,58 @@
+# AWS FreeRTOS Port
+
+## The Porting Process
+
+Porting from AWS FreeRTOS applications can be done using the following sections. There are several topics to cover including building, porting, authentication, and feature set.
+
+## Building
+
+Like the AWS FreeRTOS library, we use CMake to build the middleware. You may use the directions [here](https://github.com/Azure/azure-iot-middleware-freertos#using-cmake) to correctly set up the build environment for your project to generate and compile.
+
+To see the detailed diff showing the changes in the CMake to get the sample building with the Azure IoT middleware for FreeRTOS, please see [here](https://github.com/aws/amazon-freertos/commit/cc30c2493f13b441d3f4029aacac420756cc09f9).
```

Comment:
https://github.com/aws/amazon-freertos/commit/cc30c2493f13b441d3f4029aacac420756cc09f9 

this only talks about stm32l475-disco, can we emphasis that we are talking about this board.... this guide is generic with one off example